HorizonEditor/CMakeLists.txt

```diff
@@ -78,4 +78,22 @@ add_custom_command(
 	ARGS -E copy_directory
 	${CMAKE_CURRENT_LIST_DIR}/Scripts
 	$<TARGET_FILE_DIR:HznEditor>/Scripts
+)
+
+add_custom_command(
+	TARGET HznEditor
+	POST_BUILD
+	COMMAND ${CMAKE_COMMAND}
+	ARGS -E copy
+	${CMAKE_CURRENT_LIST_DIR}/premake5.exe
+	$<TARGET_FILE_DIR:HznEditor>/premake5.exe
+)
+
+add_custom_command(
```

Comment:
Adding this block in twice? Why?

Reply:
One is for the script file, and the other is for the premake application that runs the script file.